### DISCUSSION TITLE
EventHandler problem & API documentation

### DISCUSSION THREAD

**Discussion by MyMailHasNoDot** [LilyBukkit/LilyBukkit-QA#49] [Q&A]
1. I can't see EventHandler annotation. Where is it? If not, what is the alternative?
I looked in `org.bukkit.event` but nothing.

2. Can you complete the API documentation?

**Comment by Vladg24YT** [comment]
Hello. 

The LilyBukkit-QA server core is currently being rewritten from scratch at [LilyBukkit-3 repository](https://github.com/LilyBukkit/LilyBukkit-3/tree/QA-build). It is being developed as a fork of CraftBukkit rather than a custom implementation, thus it **will** have event handling as well as some other nuances that I missed during original LB-QA development.

API documentation is currently not being worked on due to the limited time I can allocate to working on this project.
